.github/workflows/maven.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
```

Comment:
Why JDK 11 if there is java-version 8 on the following line? I think using Java 8 is fine -- we use Java 8 for the nightly/official builds as well (some of the users do still use Java 8 LTS).

Reply:
fixed